staff/mgarciarojas/Javascript exercises/maxFunction.js

```diff
@@ -0,0 +1,11 @@
+//max() function
+//Define a function max() that takes two numbers as arguments and returns
+// the largest of them. Use the if-then-else construct available in Javascript.
+
+function max (a, b){
+if (a > b){
+console.log(a)
```

Comment:
instead of executing `console.log` inside the functions, it's better that functions return values

Reply:
Done